docs/topics/compatibility-guide-19.md

```diff
@@ -148,7 +138,7 @@ perspective
 > **Incompatible change type**: source
 > 
 > **Short summary**: Kotlin will report a compile error when an explicit or implicit super call is delegated 
-> to an _abstract_ member of the superclass, even if there's a default implementation in a super interface
+> to an _abstract_ member of the superclass, even if there's a default implementation in a super interface.
```

Comment:
Yes I think so. If it's just one sentence you can get away with not having a period at the end. But when it is more than one sentence you really should use them consistently. Since the short summaries are sometimes one or two sentences I added the periods for every short summary for consistency.

Reply:
ok